src/DefaultProgressBar.jsx

```diff
@@ -28,6 +28,7 @@ function DefaultProgressBar({
   theme,
   orientation,
   disabled,
+  style,
```

Comment:
also here

Reply:
This is a rheostat-specific prop that predates RWS. Its contents also need to be flipped for RTL (notably this style prop includes the dynamically calculated left and right positions)